core/jvm/src/test/scala/scalaz/zio/FiberSpec.scala

```diff
@@ -0,0 +1,23 @@
+package scalaz.zio
+
+class FiberSpec(implicit ee: org.specs2.concurrent.ExecutionEnv) extends TestRuntime {
+  def is =
+    "FiberSpec".title ^ s2"""
+    Create a new Fiber and
+      lift it into Managed $e1
+    """
+
+  def e1 = {
```

Comment:
Something like this could probably work::
```scala
  for {
      ref <- Ref.make(false)
      fib <- IO.never.ensuring(ref.set(true)).fork
      _   <- Managed.make(IO.succeed(fib))(_.interrupt).use(_ => IO.unit)
      res <- ref.get
    } yield res must_=== true
```

Reply:
Yes, it is cleaner. Still not hitting all the cases I thought about, but once I rethink it, it is ok. Pushed the changes a moment ago.